src/applications/mhv/secure-messaging/components/OlderMessages.jsx

```diff
@@ -1,22 +1,77 @@
-import React from 'react';
-import { VaAdditionalInfo } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import React, { useState } from 'react';
 import Message from './Message';
 
-const OlderMessages = () => (
-  <div>
-    <VaAdditionalInfo
-      trigger="Older messages in this conversation"
-      disable-border
-    >
-      <VaAdditionalInfo
-        class="expand-messages"
-        trigger="Expand All Messages"
-        disable-border
-      />
+const OlderMessages = () => {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [isExpandedAll, setIsExpandedAll] = useState(false);
 
-      <Message />
-    </VaAdditionalInfo>
-  </div>
-);
+  const handleExpandOlder = e => {
+    // prevent fromp expanding/collapsing on Tab key press for accessibility
+    if (e.key !== 'Tab') {
+      setIsExpanded(!isExpanded);
+      if (isExpandedAll) {
+        // if collapsing Older Messages, collapse all expanded messages below as well
+        setIsExpandedAll(!isExpandedAll);
+      }
+    }
+  };
+
+  const handleExpandAll = e => {
+    // prevent fromp expanding/collapsing on Tab key press for accessibility
+    if (e.key !== 'Tab') {
+      setIsExpandedAll(!isExpandedAll);
+    }
+  };
+
+  return (
+    <div className="older-messages">
+      <div

```

Comment:
Consider adding `aria-expanded`/`aria-controls` here

Reply:
added as well. Thank you for pointing that out. New to aria and accessibility attributes, pardon any inconsistencies